net/tcp/tcp.h

```diff
@@ -86,6 +86,15 @@
 #  endif
 #endif
 
+/* 32-bit modular arithmetics for tcp sequence numbers */
+
+#define TCP_SEQ_LT(a, b)	((int32_t)((a) - (b)) < 0)
+#define TCP_SEQ_GT(a, b)	TCP_SEQ_LT(b, a)
+#define TCP_SEQ_LTE(a, b)	!TCP_SEQ_GT(a, b)
```

Comment:
May I suggest:

`#define TCP_SEQ_LTE(a, b)	(!TCP_SEQ_GT(a, b))`

(Added parenthesis around the definition.)

Reply:
ok

Reply:
done